images/entrypoint.sh

```diff
@@ -134,6 +142,35 @@ fi
 
 # ---------------------- end Generate a "kube-config".
 
+# ------------------------------- Generate "00-multus.conf"
+
+if [ "$MULTUS_CONF_FILE" == "auto" ]; then
+  echo "Generating Multus configuration file ..."
+  MASTER_PLUGIN="$(ls $CNI_CONF_DIR | grep -E '\.conf(list)?$' | head -1)"
+  if [ "$MASTER_PLUGIN" == "" ]; then
+    echo "Error: Multus could not be configured: no master plugin was found."
+    exit 1;
+  elif [ "$MASTER_PLUGIN" == "00-multus.conf" ]; then
+    echo "Warning: Multus is already configured: auto configuration skipped."
+  else
+    MASTER_PLUGIN_JSON="$(cat $CNI_CONF_DIR/$MASTER_PLUGIN)"
+    CONF=$(cat <<-EOF
+			{
+				"name": "multus-cni-network",
+				"type": "multus",
+				"kubeconfig": "$MULTUS_KUBECONFIG",
```

Comment:
This will generate ```Err in loading K8s Delegates k8s args: GetK8sClient: failed to get context for the kubeconfig /host/etc/cni/net.d/multus.d/multus.kubeconfig``` it should be ```etc/cni/net.d/multus.d/multus.kubeconfig```